packages/@react-spectrum/table/src/TableView.tsx

```diff
@@ -929,6 +1041,33 @@ function TableSelectAllCell({column}) {
   );
 }
 
+function TableDragHeaderCell({column}) {
+  let ref = useRef();
+  let {state} = useTableContext();
+  let {columnHeaderProps} = useTableColumnHeader({
+    node: column,
+    isVirtualized: true
+  }, state, ref);
+  // TODO: How do we skip navigation to this cell?
```

Comment:
i think it's ok to have a blank table drag header cell with an aria-label that is navigable